error/error.go

```diff
@@ -67,15 +67,15 @@ func ParseLevel(level string) (Level, error) {
 	case "SHOULDNOT":
 		fallthrough
 	case "RECOMMENDED":
-		fallthrough
-	case "NOTRECOMMENDED":
 		return Should, nil
+	case "NOTRECOMMENDED":
+		fallthrough
```

Comment:
I don't think we want to make this change. As it stands in master, `NOTRECOMMENDED` returns `Should`.  That's [the appropriate level for `NOTRECOMMENDED`][1].  With your change, `NOTRECOMMENDED` would return `Must`, and that's too strong.

[1]: https://tools.ietf.org/html/rfc2119#section-4